src/ventus/wm/window.js

```diff
@@ -322,13 +324,23 @@ function(Emitter, View, WindowTemplate) {
 					this.el.hide();
 
 					// Remove element
-					this.$content.html('');
+					if(!this.softRemove) {
```

Comment:
I think this would be better if we make the distinction between 'close' and 'destroy'. I think that would be a better solution to this, avoiding to add a new option.